docs/core/tools/dotnet-publish.md

```diff
@@ -65,6 +65,10 @@ Configuration to use when building the project. The default value is `Debug`.
 
 Defines the version suffix to replace the asterisk (`*`) in the version field of the project file.
 
+`--manifest`
+
+Specifies one or several [target manifests](../deploying/runtime-package-store.md) to use to trim the set of packages that will be published with the application.
```

Comment:
Not really. For example -s for dotnet restore, you actually have to repeat the option again. I don't think it hurts to tell customers how the input is supposed to be provided. For dotnet sln, we show in the syntax that is one after the other.